lib/index.ts

```diff
@@ -67,7 +74,7 @@ class PWMetrics {
     this.logger = Logger.getInstance({showOutput: this.flags.showOutput});
   }
 
-  async start() {
+  async start(outputDataCallback) {
```

Comment:
I'm ok with cb for now. wdyt if in future we make `outputDataCallback` like method of PWMetrics class and drop code from cli.js. It can be more clear to read and logic for handling flags will be placed in one class?

Reply:
Sure! I didn't put that code in the class because I thought it was there, in the promise chain, on purpose. I'll do that.

Reply:
Done in https://github.com/paulirish/pwmetrics/pull/195